translations/bulk_import_translations.py

```diff
@@ -38,22 +39,32 @@ def bulk_add(translations):
             if ref[0] == 'programs_program':
                 try:
                     obj = Program.objects.get(external_name=ref[1])
+                    obj.active = True
                 except ObjectDoesNotExist:
-                    raise Exception(f'Program with expternal name of {ref[1]} does not exist. Please add it.')
-                obj.active = True
+                    obj = Program.objects.new_program(ref[1])
+                    obj.external_name = ref[1]
                 obj.save()
-            if ref[0] == 'programs_navigator':
+            elif ref[0] == 'programs_navigator':
                 try:
                     obj = Navigator.objects.get(external_name=ref[1])
                 except ObjectDoesNotExist:
-                    raise Exception(f'Navigator with expternal name of {ref[1]} does not exist. Please add it.')
-            if ref[0] == 'programs_urgentneed':
+                    obj = Navigator.objects.new_navigator(ref[1], None)
+                    obj.external_name = ref[1]
+                    obj.save()
+            elif ref[0] == 'programs_urgentneed':
                 try:
                     obj = UrgentNeed.objects.get(external_name=ref[1])
+                    obj.active = True
                 except ObjectDoesNotExist:
-                    raise Exception(f'Urgent Need with expternal name of {ref[1]} does not exist. Please add it.')
-                obj.active = True
+                    obj = UrgentNeed.objects.new_urgent_need(ref[1], None)
+                    obj.external_name = ref[1]
                 obj.save()
+            elif ref[0] == 'programs_document':
+                try:
+                    obj = Document.objects.get(external_name=ref[1])
+                except ObjectDoesNotExist:
+                    obj = Document.objects.new_document(ref[1])
+
```

Comment:
I had to call `obj.save()` on some of the other ones because I was chnaging one of the fields. For example `obj.external_name = ref[1]` on line 60 changes the external name, so it needs to get saved after.